google/resource_compute_instance_test.go

```diff
@@ -873,7 +874,7 @@ func testAccCheckComputeInstanceDestroy(s *terraform.State) error {
 	return nil
 }
 
-func testAccCheckComputeInstanceExists(n string, instance *compute.Instance) resource.TestCheckFunc {
+func testAccCheckComputeInstanceExists(n, p string, instance *compute.Instance) resource.TestCheckFunc {
```

Comment:
Same comment than for instance template.

Reply:
Same.